lib/start.js

```diff
@@ -2,20 +2,22 @@ const openBrowser = require('react-dev-utils/openBrowser')
 const webpack = require('webpack')
 const WebpackDevServer = require('webpack-dev-server')
 const chalk = require('chalk')
+const initializeMiddleware = require('./initializeMiddleware')
+const express = require('express')
 
 const webpackConfig = require('./webpack.config')
 
 module.exports = ({ args, getPackage }) => {
+  const app = express()
   const pkg = getPackage()
-  const publicPath = args.contextPath || pkg['context-path']
+  const publicPath = pkg['context-path']
 
-  const main = args.main || pkg.main
```

Comment:
Undo, I think you might have applied changes against an older version of ace. This is a breaking change.